src/Microsoft.DotNet.ServiceFabric.ServiceHost/TemporaryFiles.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Fabric;
+using System.IO;
+
+namespace Microsoft.DotNet.ServiceFabric.ServiceHost
+{
+    public class TemporaryFiles : IDisposable
+    {
+        private string _isolatedTempPath;
+
+        public TemporaryFiles(ServiceContext context)
+        {
+            ServiceContext context1 = context;
+            _isolatedTempPath = Environment.ExpandEnvironmentVariables($"%TEMP%\\{context1.ServiceTypeName}\\{context1.ReplicaOrInstanceId}");
```

Comment:
Good point. I'll switch back to Path.GetTempPath.  No real reason not to (my previous attempt was setting %TEMP% process wide, so there was a lot more environment manipulation, but that didn't work because the SubscriptionActor and PullRequestActor are in the same process).